### ISSUE TITLE
Write README for the project

### ISSUE THREAD

**Issue by esther-poniatowski** [esther-poniatowski/multitask-context-dependent-behavior#44]
Specify : customizations of labels, non-code issues.
Implementation ideas : "Blocked by" and "Blocking" (automatically switch the start and end dates for all the children), "Supertask" (from the presence of a "TaskList" keyword in the comments section).

**Comment by esther-poniatowski** [comment]
Not a priority
